ui/page/sign_message_page.go

```diff
@@ -20,11 +21,12 @@ const SignMessagePageID = "SignMessage"
 
 type SignMessagePage struct {
 	*load.Load
-	theme     *decredmaterial.Theme
 	container layout.List
 	wallet    *dcrlibwallet.Wallet
 
 	isSigningMessage                           bool
+	isValidatedMsg                             bool
+	isValidatedAddress                         bool
```

Comment:
Proper name should be `hasValidatedMsg` or `addressIsValid`

Reply:
These two variables are modified in several references but only used once and they were used together. Instead of two variables, make it one and it should be `true` if message and address is valid.